.jshintrc

```diff
@@ -0,0 +1,24 @@
+{
+	  "node": true,
+	  "esnext": true,
+	  "bitwise": true,
+	  "camelcase": true,
+	  "curly": true,
+	  "eqeqeq": true,
+	  "immed": true,
+	  "indent": 2,
+	  "newcap": true,
+	  "noarg": true,
+	  "quotmark": "double",
```

Comment:
Change the quotmark to `single`
```
"quotmark": "single",
```
It's recommended by Andela JavaScript Style Guide.